bool visible
float64 id
bool is_left
bool is_right
bool is_valid
float64 grab_strength
float64 pinch_strength
float64 confidence
float64[3] palm_normal
float64[3] direction
float64[3] palm_position

teleop_gesture_toolbox/Bone[20] finger_bones

float64[3] palm_velocity
float64[9] basis
float64 palm_width
float64[3] sphere_center
float64 sphere_radius
float64[3] stabilized_palm_position
float64 time_visible

float64[3] wrist_position
float64[3] elbow_position

bool arm_valid
float64 arm_width
float64[3] arm_direction
float64[9] arm_basis
